scripts/bootstrap.sh

```diff
@@ -157,8 +157,8 @@ image_load() {
       #TODO Is this install directory parameterized?
       IMAGE_DIR=/opt/ibm/cluster/images
       mkdir -p ${IMAGE_DIR}
-      ${awscli} s3 cp ${image_location} ${IMAGE_DIR}/ibm-cloud-private-x86_64-3.1.0.tar.gz 
-      tar zxf ${IMAGE_DIR}/ibm-cloud-private-x86_64-3.1.0.tar.gz -O | docker load
+      ${awscli} s3 cp ${image_location} ${IMAGE_DIR}/ibm-cloud-private-x86_64-3.1.2.tar.gz 
+      tar zxf ${IMAGE_DIR}/ibm-cloud-private-x86_64-3.1.2.tar.gz -O | docker load
```

Comment:
this was meant as a temporary filename and it didn't really matter what the version on this was as it would just extract whatever was in S3 and load it.

if you want, you can change this to `${IMAGE_DIR}/`basename ${image_location}` so it just uses the filename of what was out there on s3.  so we don't have to change the filenames every time there's a new release

Reply:
+1 to having those two lines say
```
      ${awscli} s3 cp ${image_location} ${IMAGE_DIR}/$(basename ${image_location})
      tar zxf ${IMAGE_DIR}/$(basename ${image_location}) -O | docker load
```

Will both be more future proof and avoid confusion